src/bindings/js/node/tests/basic.test.js

```diff
@@ -10,10 +10,84 @@ const { getModelPath } = require('./utils.js');
 let testXml = getModelPath();
 const core = new ov.Core();
 const model = core.readModelSync(testXml);
-const compiledModel = core.compileModel(model, 'CPU');
+const compiledModel = core.compileModelSync(model, 'CPU');
 const modelLike = [[model],
   [compiledModel]];
 
+describe('Core.compileModelSync()', () => {
+  const tput = {'PERFORMANCE_HINT': 'THROUGHPUT'};
+  
+  it('compileModelSync(model:Model, deviceName: string, config: {}) ', () => {
+    const cm = core.compileModelSync(model, 'CPU', tput);
+    assert.deepStrictEqual(cm.output(0).shape, [1, 10]);
+  });
+  
+  it('compileModelSync(model:model_path, deviceName: string, config: {}) ', () => {
+    const cm = core.compileModelSync(testXml, 'CPU', tput);
+    assert.equal(cm.inputs.length, 1);
+  });
+
+  it('compileModelSync(model:model_path, deviceName: string) ', () => {
+    const cm = core.compileModelSync(testXml, 'CPU');
+    assert.deepStrictEqual(cm.output(0).shape, [1, 10]);
+  });
+
+  it('compileModelSync(model, device, config) throws when config is a string', () => {
+    assert.throws(
+      () => core.compileModelSync(model, 'CPU', 'string'),
+      /Cannot convert Napi::Value to std::map<std::string, ov::Any>/
+    );
+  });
+
+  it('compileModelSync(model, device, config) throws when config value is not a string', () => {
+    assert.throws(
+      () => core.compileModelSync(model, 'CPU', {"PERFORMANCE_HINT":tput}),
+        /Cannot convert Napi::Value to ov::Any/
+    );
+  });
+
+} );
+
+describe('Core.compileModel()', () => {
+    const tput = {'PERFORMANCE_HINT': 'THROUGHPUT'};
+    
+    it('compileModel(model:Model, deviceName: string, config: {}) ', () => {
+      core.compileModel(model, 'CPU', tput).then(cm => {
+        assert.deepStrictEqual(cm.output(0).shape, [1, 10]);
+      });
+      
+    });
+    
+    it('compileModel(model:model_path, deviceName: string, config: {}) ', () => {
+      core.compileModel(testXml, 'CPU', tput).then(cm => {
+        assert.equal(cm.inputs.length, 1);
+      })
+      
+    });
+  
+    it('compileModel(model:model_path, deviceName: string) ', () => {
+      core.compileModel(testXml, 'CPU').then(cm => {
+        assert.deepStrictEqual(cm.output(0).shape, [1, 10]);
+      });
+      
+    });
+  
+    it('compileModel(model, device, config) throws when config is a string', () => {
+      assert.throws(
+        () => core.compileModel(model, 'CPU', 'string').then(),
+        /Cannot convert Napi::Value to std::map<std::string, ov::Any>/
+      );
+    });
+  
+    it('compileModel(model, device, config) throws when config value is not a string', () => {
```

Comment:
Line 75: 'throws when config is a string'
Line 82: 'throws when config value is not a string'

These confusing me

Reply:
Line 75: It means that config has to be an object
Line 82: config object values have to strings